docs/index.rst

```diff
@@ -19,6 +19,9 @@ Welcome to SiliconCompiler's Documentation!
    Advanced Guide <development_guide/index>
    reference_manual/index
 
+**Version**:
+|version|
+
```

Comment:
maybe we should also add this to the footer in the docs so it's available on every page.